src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,46 @@
 layout: content.html
 ---
 
+<%
+const require = this.process.mainModule.require
+const fs = require('fs')
+%>
+
 # Plugins
-*Seneca* plugins enable you to build awesome microservices, incredibly fast.
+
+_Seneca_ plugins enable you to build awesome microservices, incredibly fast.
 Some plugins are maintained in-house, while others are provided by the community.
 
 On this page, we provide an `npm` link and total downloads per month for any plugins
 we find or publish ourselves, so you only need to look in one place.
 
+<!-- <%
+let testFile = JSON.parse(fs.readFileSync('./assets/files/plugins.json', 'utf8'))
```

Comment:
we won't use JSON - see above, but if we did, note that `require('a-json-file.json')` will "just work" - no JSON.parse needed

Reply:
Will keep in mind, thanks